packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -108,6 +109,17 @@ export const getDocs: () => JSX.Element = () => {
         tutorial. https://docs.microsoft.com/en-us/azure/communication-services/tutorials/file-sharing-tutorial
       </Description>
       <Source code={addFileSharingSnippet} />
+
+      <Heading>PSTN and 1:N Calling</Heading>
+      <SingleLineBetaBanner />
+      <MessageBar>Note: see CallComposite for detailed implementation description</MessageBar>
+      <Description>
+        The CallWithChatComposite supports the PSTN Calling modality as well. With CallWithChat you will need to still
+        provide a chat thread Id for instances where you will add a Azure Communications user to the call. If you are
+        looking to use the CallWithChatComposite and are looking to hide the chat because you are only calling PSTN
```

Comment:
I would leave this out with the assumption that if someone is doing a call only they should be using the call composite

Reply:
Oh actually I guess it could be dynamically hidden if suddenly an ACS user is there or not